component/pyroscope/scrape/scrape_loop.go

```diff
@@ -238,7 +238,7 @@ func (t *scrapeLoop) scrape() {
 	if len(b) > 0 {
 		t.lastScrapeSize = len(b)
 	}
-	if err := t.appender.Append(context.Background(), t.labels, []*pyroscope.RawSample{{RawProfile: b}}); err != nil {
+	if err := t.appender.Append(context.Background(), t.privateLabels, []*pyroscope.RawSample{{RawProfile: b}}); err != nil {
```

Comment:
Maybe I don't have a full context here, but shouldn't we call `Append` with either just public labels, or with all public + private labels (which seems to be the case previously?)

Reply:
Traced the code and I see that this matches the previous behaviour now.